mmv1/third_party/terraform/services/apphub/data_source_apphub_discovered_workload_test.go

```diff
@@ -0,0 +1,112 @@
+package apphub_test
+
+import (
+    "testing"
+
+    "github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+    "github.com/hashicorp/terraform-provider-google/google/acctest"
+)
+
+func TestAccDataSourceApphubDiscoveredWorkload_basic(t *testing.T) {
+    t.Parallel()
+
+    context := map[string]interface{}{
+    	"random_suffix": acctest.RandString(t, 10),
+    }
+
+    acctest.VcrTest(t, resource.TestCase{
+        PreCheck:                 func() { acctest.AccTestPreCheck(t) },
+        ProtoV5ProviderFactories: acctest.ProtoV5ProviderFactories(t),
+        Steps: []resource.TestStep{
+            {
+                Config: testDataSourceApphubDiscoveredWorkload_basic(context),
+            },
+        },
+    })
+}
+
+func testDataSourceApphubDiscoveredWorkload_basic(context map[string]interface{}) string {
+    return acctest.Nprintf(`
+data "google_apphub_discovered_workload" "catalog-workload" {
+  provider = google
+  location = "us-central1"
+  workload_uri = "${replace(google_compute_region_instance_group_manager.mig.instance_group, "https://www.googleapis.com/compute/v1", "//compute.googleapis.com")}"
```

Comment:
https://registry.terraform.io/providers/hashicorp/google/latest/docs/data-sources/compute_region_instance_group has an example of how we could fetch the underlying instance group uri without having to construct the string.

Reply:
Done.